apps/quarkus-mp-orm-dbs-awt/src/main/resources/application.properties

```diff
@@ -17,7 +17,7 @@ quarkus.datasource.db2.username=quarkus
 quarkus.datasource.db2.password=quarkus
 quarkus.datasource.db2.devservices.enabled=true
 quarkus.datasource.db2.devservices.port=49157
-quarkus.datasource.db2.devservices.image-name=docker.io/library/mariadb:10.11
+quarkus.datasource.db2.devservices.image-name=quay.io/mariadb-foundation/mariadb-devel:10.11
```

Comment:
I guess that's needed for debugging purposes? Aren't the regular logs enough?

Reply:
@zakkak As a matter of fact it is due to DockerHub rate limiting and this being a quay.io mariadb from a publisher I decided to trust...